server/src/main/java/com/cloud/server/ManagementServerImpl.java

```diff
@@ -1011,6 +1015,8 @@ public class ManagementServerImpl extends ManagerBase implements ManagementServe
     UserDataManager userDataManager;
     @Inject
     StoragePoolTagsDao storagePoolTagsDao;
+    @Inject
+    protected ManagementServerJoinDao _managementServerJoinDao;
```

Comment:
```suggestion
    protected ManagementServerJoinDao managementServerJoinDao;
```
The underscore at the start is a legacy naming "convention" that is not used by the project anymore. The current convention is to use plain camelCase https://cwiki.apache.org/confluence/display/CLOUDSTACK/Coding+conventions